server/core/src/main/java/dev/slimevr/config/TapDetectionConfig.kt

```diff
@@ -14,20 +14,16 @@ class TapDetectionConfig {
 	var mountingResetEnabled = true
 	var setupMode = false
 	var yawResetTaps = 2
-		// clamp to 2-3 to prevent errors
 		set(yawResetTaps) {
 			field = FastMath.clamp(yawResetTaps.toFloat(), 2f, 10f).toInt()
-			field = yawResetTaps
 		}
 	var fullResetTaps = 3
 		set(fullResetTaps) {
 			field = FastMath.clamp(fullResetTaps.toFloat(), 2f, 10f).toInt()
-			field = fullResetTaps
 		}
 	var mountingResetTaps = 3
 		set(mountingResetTaps) {
 			field = FastMath.clamp(mountingResetTaps.toFloat(), 2f, 10f).toInt()
```

Comment:
We can also just use `coerceIn` for these to not convert to floats and stuff
See: https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.ranges/coerce-in.html
(Might need an import)

```suggestion
			field = yawResetTaps.coerceIn(2, 10)
		}
	var fullResetTaps = 3
		set(fullResetTaps) {
			field = fullResetTaps.coerceIn(2, 10)
		}
	var mountingResetTaps = 3
		set(mountingResetTaps) {
			field = mountingResetTaps.coerceIn(2, 10)
```